examples/cifar10.py

```diff
@@ -421,11 +421,11 @@ def parse_args():
         "using Data Parallel or Distributed Data Parallel",
     )
     parser.add_argument(
-        "--sample-rate",
-        default=0.04,
+        "--batch-size",
+        default=2000,
         type=float,
-        metavar="SR",
-        help="sample rate used for batch construction (default: 0.005)",
+        metavar="N",
+        help="train bacth size",
```

Comment:
may be change this to approximate batch size in the help string, since the batch size will not necessarily be 2K.